pkg/profiling/config/config.go

```diff
@@ -2,10 +2,9 @@ package config
 
 import (
 	"io/ioutil"
-	"os"
 	"path/filepath"
-	"strings"
 
+	"gopkg.in/yaml.v2"
```

Comment:
Just something to call out - you _may_ run into weird serialization errors using this yaml package, namely around handling of default values which I ran into recently between the sno/rukpak work. Might be worth using `k8s.io/apimachinery/pkg/util/yaml` instead as I've had more reliable results using that package.